### PR TITLE
Dialogs

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
This PR adds header icon to the Dialog component and refactors Share, Help and Open dialogs to include the header icon.

### CODE REVIEW

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter PTAL

File: src/Components/Dialog.jsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React from 'react'
+import Box from '@mui/material/Box'
 import Button from '@mui/material/Button'
 import DialogActions from '@mui/material/DialogActions'
 import DialogContent from '@mui/material/DialogContent'
 import DialogTitle from '@mui/material/DialogTitle'
 import MuiDialog from '@mui/material/Dialog'
+import Typography from '@mui/material/Typography'
 import {assertDefined} from '../utils/assert'
```
Comment (pablo-mayrgundter) [inline src/Components/Dialog.jsx:9]: imports: package before file, code before svg

File: src/Components/HelpControl.jsx
Comment (pablo-mayrgundter) [inline src/Components/HelpControl.jsx:1]: imports: package before file, code before svg

File: src/Components/OpenModelControl.jsx
Comment (pablo-mayrgundter) [inline src/Components/OpenModelControl.jsx:1]: imports: package before file, code before svg

File: src/Components/OperationsGroup.jsx [hunk 2]
```diff
@@ -110,9 +110,7 @@ export default function OperationsGroup({deselectItems}) {
           />
       }
       {isCollaborationGroupVisible &&
-        <Box sx={{marginTop: '8px'}}>
```
Comment (pablo-mayrgundter) [inline src/Components/OperationsGroup.jsx:113]: can you do em instead of px?  px doesn't change with type size

File: src/Components/ShareControl.jsx
Comment (pablo-mayrgundter) [inline src/Components/ShareControl.jsx:1]: imports: package before file, code before svg

**Review by pablo-mayrgundter — APPROVED** [review]
Pls fix style issues

File: src/Components/Dialog.jsx (hunk 1 above)
Comment (OlegMoshkovich) [reply src/Components/Dialog.jsx:9]: yep!

**Pull request merged**